components/awards/ScoreBasedAwards.js

```diff
@@ -56,10 +56,15 @@ export default function Performances({ award, session }) {
   const getPerformances = async () => {
     setLoading(true);
 
+    let performanceUrl = `/api/performances/collect?eventID=${event.id}`;
+    if (award.awards_categories && award.awards_categories.length !== 0) {
+      performanceUrl += `&settingIDs=${award.awards_categories.join(',')}`;
```

Comment:
If it's possible to know which element in the awards_category array corresponds to, I can do 3 query parameters ex. competition_level, dance_size, dance_style. Is that possible to know?

Reply:
There is a setting type which you can look up with a corresponding SettingType enum.